src/primitives/block.cpp

```diff
@@ -25,9 +25,8 @@ std::string CBlock::ToString() const
         hashMerkleRoot.ToString(),
         nTime, nBits, nNonce,
         vtx.size());
-    for (unsigned int i = 0; i < vtx.size(); i++)
-    {
-        s << "  " << vtx[i]->ToString() << "\n";
+    for (const auto &tx : vtx) {
```

Comment:
Yeah, I was unsure about that. It seems counter intuitive to me since it is the variable that is a reference, not the type.

Anyway, I'll look into it so that it is consistent with the rest of the core base.

Thanks.